cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/handler/async/runnable/LoadRunnable.java

```diff
@@ -64,13 +66,29 @@ protected BrokerStats getResult() throws Exception {
         return cachedBrokerStats;
       }
     }
+    if (_populateDiskInfo && !isClusterUsingJBOD()) {
```

Comment:
Can we avoid checking `_populateDiskInfo` here again -- i.e. can't we do the following?
```
} else if (!isClusterUsingJBOD()) {
   ...
}
```

Reply:
Good idea!